src/platform/packages/shared/kbn-security-hardening/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
```

Comment:
This `BUILD.bazel` file is required in order to use this package within `kbn-ui-shared-deps-src`.